doc/21-development.md

```diff
@@ -1151,6 +1151,7 @@ General:
 - [function_types](https://www.boost.org/doc/libs/1_66_0/libs/function_types/doc/html/index.html) (header only)
 - [circular_buffer](https://www.boost.org/doc/libs/1_66_0/doc/html/circular_buffer.html) (header only)
 - [math](https://www.boost.org/doc/libs/1_66_0/libs/math/doc/html/index.html) (header only)
+- [stacktrace](https://www.boost.org/doc/libs/1_66_0/doc/html/stacktrace.html) (header only)
```

Comment:
Did you require this Boost lib? If yes, did you let CMake enforce this requirement?

Reply:
None of the other header-only libraries is currently enforced by CMake: https://github.com/Icinga/icinga2/blob/ff5385bbb5f0ab3f32b52ec4cdf94b4c1a452972/CMakeLists.txt#L166

Reply:
Ah, that's a header-only one. Fine.